hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/CallEvmTxProcessor.java

```diff
@@ -128,7 +128,11 @@ protected MessageFrame buildInitialFrame(
             final Address to,
             final Bytes payload,
             final long value) {
-        final var code = codeCache.getIfPresent(aliasManager.resolveForEvm(to));
+        final var resolvedForEvm = aliasManager.resolveForEvm(to);
+        final var code =
+                aliasManager.isMirror(resolvedForEvm)
```

Comment:
Similarly not supporting code fetching from "mirror" addresses is a behavior change, and should be preserved in v0.30 EVMs.

Reply:
Added version check.